maas/plugins/galera_check.py

```diff
@@ -82,25 +86,35 @@ def print_metrics(replica_status):
            replica_status['wsrep_local_state_uuid'])
     metric('wsrep_local_state_comment', 'string',
            replica_status['wsrep_local_state_comment'])
+    metric('max_configured_connections', 'int64',
+           replica_status['max_connections'], 'connections')
+    metric('current_connections', 'int64',
+           replica_status['Threads_connected'], 'connections')
+    metric('max_seen_connections', 'int64',
+           replica_status['Max_used_connections'], 'connections')
+    metric('percentage_used_connections', 'int64',
```

Comment:
The other metrics track it. An alarm can alert on it. I would expect the visualisation software to allow for the data to be manipulated. My thinking is that the plugins should only be responsible for data collection.